backend/api/v2beta1/pipeline.proto

```diff
@@ -95,6 +95,14 @@ service PipelineService {
     };
   }
 
+  // Creates a new pipeline and a new pipeline version in a single transaction.
+  rpc CreatePipelineAndVersion(CreatePipelineAndVersionRequest) returns (Pipeline) {
+    option (google.api.http) = {
+      post: "/apis/v2beta1/pipelines/create"
```

Comment:
This feels anti-pattern to include the verb in the path, should it be `"/apis/v2beta1/pipelineandversion"` instead?